Unlimited SNI bindings in App Service

kaushalp opened the pull request:
In App Service, when running on Basic or higher pricing tiers, customers can create unlimited number of SNI SSL Bindings. This change was made some time back. Refer the App Services pricing page here: https://azure.microsoft.com/en-us/pricing/details/app-service/#ssl-connections

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

kaushalp committed:
Unlimited SNI bindings in App Service

In App Service, when running on Basic or higher pricing tiers, customers can create unlimited number of SNI SSL Bindings. This change was made some time back. Refer the App Services pricing page here: https://azure.microsoft.com/en-us/pricing/details/app-service/#ssl-connections
